### PR TITLE
MULE-19092: fix munit tests

### PR DESCRIPTION
**Pull request by jcgrethe**

### CODE REVIEW

File: core-tests/src/test/java/org/mule/runtime/core/internal/processor/interceptor/ReactiveInterceptorAdapterTestCase.java [hunk 1]
```diff
@@ -205,6 +206,7 @@ public void interceptorApplied() throws Exception {
 
       assertThat(((InternalEvent) result).getInternalParameters().entrySet(), hasSize(0));
       verifyParametersResolvedAndDisposed(times(1));
+      verify(interceptor, atLeastOnce()).getClass().getClassLoader();
```
Comment (elrodro83) [inline core-tests/src/test/java/org/mule/runtime/core/internal/processor/interceptor/ReactiveInterceptorAdapterTestCase.java:209]: create a different set of test cases to assert this specifically,

Comment (elrodro83) [inline core-tests/src/test/java/org/mule/runtime/core/internal/processor/interceptor/ReactiveInterceptorAdapterTestCase.java:209]: i think this is just testing that getClass was called, but not getClassloader. Is there a more expressive way to verify the expected behavior?

**Review by elrodro83 — CHANGES_REQUESTED** [review]

**Comment by jcgrethe** [comment]
--validate

**Comment by jcgrethe** [comment]
--validate

**Comment by jcgrethe** [comment]
--validate

**Pull request merged**